src/Symfony/Component/Translation/Bridge/Loco/composer.json

```diff
@@ -20,6 +20,12 @@
         "symfony/http-client": "^5.3",
         "symfony/translation": "^5.3"
     },
+    "require-dev": {
+        "symfony/config": "^4.4|^5.0"
+    },
+    "conflict": {
+        "symfony/config": "<4.4"
```

Comment:
I don't think we need this conflict. The bridge itself does not use the config component. Its testsuite uses features of `symfony/translation` needing it, and so the conflict rule is managed in `symfony/translation`

Reply:
You are correct. Removed. 🙂